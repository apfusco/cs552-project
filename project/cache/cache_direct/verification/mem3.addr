0 1 0 0 
0 1 2 0
0 1 4 0
0 1 6 0
0 1 2048 0
0 1 2050 0
0 1 2052 0
0 1 2054 0
