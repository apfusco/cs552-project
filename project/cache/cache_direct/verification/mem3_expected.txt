0 1 0 0 --> MISS
0 1 2 0 --> HIT
0 1 4 0 --> HIT
0 1 6 0 --> HIT
0 1 2048 0 --> MISS (eviction)
0 1 2050 0 --> HIT
0 1 2052 0 --> HIT
0 1 2054 0 --> HIT
